pkg/operator/controller/extension/admission/admission.go

```diff
@@ -252,6 +272,14 @@ func virtualDeploymentSpecified(extension *operatorv1alpha1.Extension) bool {
 		extension.Spec.Deployment.AdmissionDeployment.VirtualCluster.Helm != nil
 }
 
+func virtualNamespace(extension *operatorv1alpha1.Extension) *corev1.Namespace {
+	return &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: fmt.Sprintf("extension-%s", extension.Name),
```

Comment:
Should we add a some type of documentation annotation that explains the purpose of the otherwise empty namespace? It could be helpful for operators.

Reply:
Now, the `extensions.operator.gardener.cloud/name` label is set to the name of the extension.